src/TraceEvent/TraceEventSession.cs

```diff
@@ -2817,27 +2845,36 @@ public enum TraceEventSessionOptions
         /// for sessions that expect more than 1K events per second.
         /// </summary>
         NoPerProcessorBuffering = 8,
+        /// <summary>
+        /// Creates a user-mode event tracing session that runs in the same process as its event trace provider.
+        /// </summary>
+        PrivateLogger = 16,
+        /// <summary>
+        /// Use in conjunction with the EVENT_TRACE_PRIVATE_LOGGER_MODE mode to start a private session.
```

Comment:
Should we switch `EVENT_TRACE_PRIVATE_LOGGER_MODE` to `PrivateLogger` since that's the value we expose here?

Reply:
Makes sense, this was the MSDN docs quoted verbatim.